htdocs/js/foundation/foundation/foundation.abide.js

```diff
@@ -4,19 +4,24 @@
   Foundation.libs.abide = {
     name : 'abide',
 
-    version : '5.4.7',
+    version : '{{VERSION}}',
```

Comment:
{{VERSION}}? (same question for all the other files)

Reply:
OH, uhhhhhhh... good catch? I'm not actually sure what this is! I SAW it, and assumed they'd changed something about how they store their version numbers, but now that I check the old v5.4.7 tag in the upstream repo, that has `{{VERSION}}` also. So, this must be an artifact of pasting from the source rather than a build. I'll go do some rebasing and see what I can learn.

Reply:
Yeah, I grabbed a build to double-check, and sure enough. Pushed a commit with the fix.